server/src/test/kotlin/org/jetbrains/bsp/bazel/server/sync/DependencyMapperTest.kt

```diff
@@ -0,0 +1,104 @@
+package org.jetbrains.bsp.bazel.server.sync
+
+import ch.epfl.scala.bsp4j.MavenDependencyModule
+import ch.epfl.scala.bsp4j.MavenDependencyModuleArtifact
+import io.kotest.matchers.shouldBe
+import org.jetbrains.bsp.bazel.bazelrunner.BazelRelease
+import org.jetbrains.bsp.bazel.server.sync.model.Label
+import org.jetbrains.bsp.bazel.server.sync.model.Library
+import org.jetbrains.bsp.bazel.server.sync.model.Module
+import org.jetbrains.bsp.bazel.server.sync.model.Project
+import org.jetbrains.bsp.bazel.server.sync.model.SourceSet
+import org.junit.jupiter.api.Test
+import java.net.URI
+import java.nio.file.Paths
+
+class DependencyMapperTest {
+
+    private val cacheLocation = "file:///home/user/.cache/bazel/_bazel_user/ae7b7b315151086e31e3b97f9ddba009/execroot/monorepo/bazel-out/k8-fastbuild-ST-4a519fd6d3e4"
+
+    @Test
+    fun `should translate dependency`() {
+        val jarUri = URI.create("$cacheLocation/bin/external/maven/org/scala-lang/scala-library/2.13.11/processed_scala-library-2.13.11.jar")
```

Comment:
does it work both with and without bzlmod?

Reply:
I will check and fix it, thanks for taking a look!

Reply:
Should work now.